<launch>
    <node name="disp"   pkg="sitl_dvrk_ros2" exec="pub_ecm_disp"   namespace="/ecm/recon3d" output="screen">
    </node>

    <node name="pclimg" pkg="sitl_dvrk_ros2" exec="pub_ecm_pclimg" namespace="/ecm/recon3d" output="screen">
    </node>

    <node name="pcl"    pkg="sitl_dvrk_ros2" exec="pub_ecm_pcl"    namespace="/ecm/recon3d" output="screen">
    </node>

</launch>